packages/blockchain-utils-validation/src/blockchains/tezos.ts

```diff
@@ -0,0 +1,64 @@
+import { AccountID } from 'caip'
+import { RemoteSigner } from '@taquito/remote-signer'
+import { char2Bytes } from '@taquito/utils'
+import { BlockchainHandler } from '../blockchain-handler'
+import { LinkProof } from '@ceramicnetwork/blockchain-utils-linking'
+import axios from 'axios'
```

Comment:
Please use `cross-fetch` as that's what we are using everywhere else.

Reply:
Got it.